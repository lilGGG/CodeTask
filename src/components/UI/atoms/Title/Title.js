import styled from 'styled-components';

const TitleStyl = styled.h1`
  font-size: 2em;
  text-align: center;
  color: chocolate;
`;

function Title({
  Text
}) {
  return (
     <> 
        <TitleStyl>
            {Text}
        </TitleStyl>
      </>
    );
}

export default Title;
