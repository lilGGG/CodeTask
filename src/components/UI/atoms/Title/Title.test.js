import React from "react";
import Title from "./Title";
import ReactDom from "react-dom";
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import { create, act } from "react-test-renderer";
import { shallow, mount, render } from "enzyme";

describe('Title test', () => {
    it('renders snapshot', () => {
      const tree = renderer.create(<Title Text="text"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });

  });
  
