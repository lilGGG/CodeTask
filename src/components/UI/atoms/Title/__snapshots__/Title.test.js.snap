// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title test renders snapshot 1`] = `
<h1
  className="sc-bczRLJ dgaykd"
>
  text
</h1>
`;
